nxviz/plots.py

```diff
@@ -250,6 +256,9 @@ def compute_node_colors(self):
         elif dtype == "continuous" and is_data_diverging(data):
             cmap = get_cmap(cmaps["diverging"].mpl_colormap)
 
+        if "cmap" in kwargs.keys():
+            cmap = get_cmap(kwargs["cmap"].mpl_colormap)
```

Comment:
Lines 259 and 260 optionally change cmap if it is set. This may override the above conditions. What are your thoughts here?

Will manually selecting a cmap result in unintended behavior that are not currently caught by the tests?

Reply:
@ahoc thanks for getting back.

Yes, please add a few tests for the different options that might show up. Rather than think of the generalized cases, having a few specific cases first is sufficient. (I don't want to over-complicate the testing up-front, we can discover the general cases later.)